File: February 2021/11.02.2021/vedat.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& A) {
         int n = A.size();
         int m = A[0].size();
        vector<vector<int>> solution(m, vector<int> (n));
        
        for(int i = 0; i < A.size(); i++){
            for(int j = 0; j < A[i].size(); j++){
                solution[j][i] = A[i][j];
            }
        }
        return solution;
    }
};

File: March 2021/12.03.2021/vedat.java
class Solution {
    public boolean hasAllCodes(String s, int k) {
        HashSet<String> nums = new HashSet<String>();
        for (int i=k; i<=s.length(); i++) {
            System.out.println(s.substring(i - k, i));
            nums.add(s.substring(i - k, i));
        }
        return nums.size() == Math.pow(2, k);
    }
}
